mergify_engine/actions/backport.py

```diff
@@ -28,17 +29,40 @@
 LOG = daiquiri.getLogger(__name__)
 
 
+def regex(value):
+    try:
+        re.compile(value)
+    except re.error as e:
+        raise voluptuous.Invalid(str(e))
+    return value
+
+
 class BackportAction(actions.Action):
-    validator = {voluptuous.Required("branches", default=[]): [str]}
+    validator = {
+        voluptuous.Required("branches", default=[]): [str],
+        voluptuous.Required("regexes", default=[]): [regex]
```

Comment:
Not a fan of the `regexes` name but I can't think of anything better.

Reply:
me neither :(

Reply:
"expressions"? Too late, I know.